Lib/test/test_json/test_tool.py

```diff
@@ -87,6 +95,7 @@ class TestMain(unittest.TestCase):
     }
     """)
 
+    @no_color
```

Comment:
Can we re-use `force_not_colorized`?

https://github.com/python/cpython/blob/0dba59e6899cd0b1717e3c1dc6110b18c91b0f8f/Lib/test/support/__init__.py#L2869

Reply:
Definitely, I didn't realize we already had this!